e2e_tests/tests/nightly/test_convergence.py

```diff
@@ -56,9 +56,9 @@ def test_mnist_estimator_accuracy() -> None:
 
 @pytest.mark.nightly  # type: ignore
 def test_mnist_pytorch_accuracy() -> None:
-    config = conf.load_config(conf.cv_examples_path("mnist_pytorch/const.yaml"))
+    config = conf.load_config(conf.cv_examples_path("mnist_multi_output_pytorch/const.yaml"))
     experiment_id = exp.run_basic_test_with_temp_config(
-        config, conf.cv_examples_path("mnist_pytorch"), 1
+        config, conf.cv_examples_path("mnist_multi_output_pytorch"), 1
```

Comment:
blocking: Looking at PR #1301, I'm pretty sure this was supposed to be tutorials_path("mnist_pytorch"), not cv_examples_path("mnist_multi_output_pytorch").

Reply:
Yes ^

Reply:
fixed